tools/topic_tools/package.xml

```diff
@@ -28,6 +28,8 @@
   <build_depend>xmlrpcpp</build_depend>
 
   <run_depend>message_runtime</run_depend>
+  <run_depend>python-numpy</run_depend>
+  <run_depend>python-texttable</run_depend>
```

Comment:
I am not sure that it is a good idea to pull in these extra dependencies in a patch release of already released stable distro (Indigo, Jade). If the API changes in `rostopic/__init__.py` get merged the command line tool itself could also be released as a standalone package?